.github/workflows/dotnet-core-desktop.yml

```diff
@@ -85,4 +84,34 @@ jobs:
       run: msbuild $env:Solution_Name /t:Restore /p:Configuration=$env:Configuration
       env:
         Configuration: ${{ matrix.configuration }}
+        
+        
+  build_ubuntu:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+
+    env:
+      Solution_Name: eawx-build.sln                         # Replace with your solution name, i.e. MyWpfApp.sln.
+      Test_Project_Path: eawx-build-test/eawx-build-test.csproj                 # Replace with the path to your test project, i.e. MyWpfApp.Tests\MyWpfApp.Tests.csproj.
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
In the PetroglyphTools repository, the `ACTIONS_ALLOW_UNSECURE_COMMANDS: true` is not needed.
That could be down to the setup though (all dotnet, and no specific versions for the actions and 3.1.x for dotnet core)